packages/shadcn/src/components/ui/radio-group.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import * as React from 'react';
+import { CheckIcon } from '@radix-ui/react-icons';
+import * as RadioGroupPrimitive from '@radix-ui/react-radio-group';
```

Comment:
Rather than use this, import each of the things you need directly from `react` and `@radix-ui/react-radio-group`